src/common/api/worker/offline/migrations/tutanota-v75.ts

```diff
@@ -0,0 +1,13 @@
+import { OfflineMigration } from "../OfflineStorageMigrator.js"
+import { OfflineStorage } from "../OfflineStorage.js"
+import { migrateAllElements } from "../StandardMigrations.js"
+import { createUserSettingsGroupRoot, UserSettingsGroupRootTypeRef } from "../../../entities/tutanota/TypeRefs.js"
+
+export const tutanota75: OfflineMigration = {
+	app: "tutanota",
+	version: 75,
+	async migrate(storage: OfflineStorage) {
+		// we need to re-initialize the UserSettingsGroupRoot to add a default value for sourceUrl on GroupSettings
+		await migrateAllElements(UserSettingsGroupRootTypeRef, storage, [createUserSettingsGroupRoot])
```

Comment:
Setting  `sourceUrl` to  `null` blindly might overwrite a non-null value set by a different client. Maybe it would be best to reload the instance?